src/filters/ws.rs

```diff
@@ -297,90 +325,78 @@ impl Message {
     /// automatically responds to the Ping messages it receives. Manual construction might still be useful in some cases
     /// like in tests or to send unidirectional heartbeats.
     pub fn pong<V: Into<Vec<u8>>>(v: V) -> Message {
-        Message {
-            inner: protocol::Message::Pong(v.into()),
-        }
+        Message::Pong(v.into())
     }
 
     /// Construct the default Close `Message`.
     pub fn close() -> Message {
```

Comment:
I'd just maintain the old behavior wdyt? [You can close the socket without it being abruptly](https://docs.rs/tungstenite/0.13.0/tungstenite/protocol/struct.WebSocket.html#method.close)